.github/workflows/ci.yml

```diff
@@ -52,6 +52,14 @@ jobs:
       - name: Install Python deps
         run: pip3 install requests wheel boto3
 
+      - name: Download kubectl and calicoctl for LKE clusters
+        run: |
+          curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
+          curl -LO "https://github.com/projectcalico/calico/releases/download/v3.25.0/calicoctl-linux-amd64"
+          chmod +x calicoctl-linux-amd64 kubectl
+          mv calicoctl-linux-amd64 /usr/local/bin/calicoctl
+          mv kubectl /usr/local/bin/kubectl
```

Comment:
Have you tried the Azure's action for installing kubectl?
https://github.com/marketplace/actions/kubectl-tool-installer